purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -105,6 +106,29 @@ class Purchases internal constructor(
             purchasesOrchestrator.updatedCustomerInfoListener = value
         }
 
+    /**
+     * Listener that receives callbacks for Customer Center events such as restore initiated,
+     * subscription cancellations, and customer feedback submission.
+     *
+     * The Customer Center is a self-service UI component that helps users manage their subscriptions
+     * and provides support features. This listener allows your app to respond to various user
+     * actions taken within the Customer Center.
+     *
+     * Take a look at [our docs] (https://rev.cat/customer-center) for more information.
+     *
+     * @important To prevent memory leaks, always set this property to null when the listener's
+     * lifecycle ends (e.g., in Activity.onDestroy() or Fragment.onDestroyView()).
+     *
+     * @note To use the Customer Center functionality, you need to include the RevenueCat UI SDK
+     * by adding the 'purchases-ui' dependency to your app's build.gradle file.
+     */
+    var customerCenterListener: CustomerCenterListener?
+        @Synchronized get() = purchasesOrchestrator.customerCenterListener
+
+        @Synchronized set(value) {
+            purchasesOrchestrator.customerCenterListener = value
+        }
```

Comment:
```suggestion
    var customerCenterListener: CustomerCenterListener? by purchasesOrchestrator::customerCenterListener
```

I think we can do without the `@Synchronized` annotations here, since the `purchasesOrchestrator::customerCenterListener` field already has those.

Reply:
oh good catch